brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1017 Fix: CF7 Styler - Message Border Width/Padding doesn't work on the front-end.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1017

### Screenshots

- https://d.pr/v/rcfCOh

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop CF7 Styler.
- Go-To Style > Success/Error Message tab.
- Add Border color, width, and padding for Success/Error Message.
- Publish the page and fill out form data and click on submit button to check Success Message changes do the same for Error Message changes. ( Keep one field empty )

### Checklist:
